.github/workflows/tests.yml

```diff
@@ -0,0 +1,18 @@
+name: Tests
```

Comment:
hey i'm not sure why the tests didn't run here, are actions enabled on this project?

Reply:
They're enabled. Maybe it needs to be merged first? I'll yolo merge and we can clean up if need be.